codalab/lib/bundle_util.py

```diff
@@ -282,19 +295,20 @@ def recurse(old_bundle_uuid):
                                         data=item2,
                                         params={'uuid': worksheet_uuid},
                                     )
-
-                            # Add the bundle item
-                            client.create(
-                                'worksheet-items',
-                                data={
-                                    'type': worksheet_util.TYPE_BUNDLE,
-                                    'worksheet': JsonApiRelationship('worksheets', worksheet_uuid),
-                                    'bundle': JsonApiRelationship('bundles', new_bundle_uuid),
-                                },
-                                params={'uuid': worksheet_uuid},
-                            )
-                            new_bundle_uuids_added.add(new_bundle_uuid)
-                            just_added = True
+                            if new_bundle_uuid not in new_bundle_uuids_added:
```

Comment:
This seems to be a bug in the original mimic code. I am seeing two bundles attached to the current worksheet. So I added logic here to check if the bundle has already been added.

Reply:
There sometimes could be multiple bundles on the same worksheet.  Can you provide the exact example / behavior?  In any case, I think we should make this change in a different PR.

Reply:
Sure. I'll create a separate PR. The behavior I saw is that when I run `cl mimic <uuid>` on a worksheet, it created two same bundles with the same content in that worksheet. I thought it should only create 1 bundle. But let me know if I misunderstood its functionality here.